helpers/paragraph.js

```diff
@@ -173,12 +165,7 @@ export function findEndOfActivePartOfNote(note: TNote): number {
       cancelledHeaderLine = i
     }
   }
-  const endOfActive =
-    (doneHeaderLine > 0)
-      ? doneHeaderLine
-      : ((cancelledHeaderLine > 0)
-        ? cancelledHeaderLine
-        : lineCount)
+  const endOfActive = doneHeaderLine > 0 ? doneHeaderLine : cancelledHeaderLine > 0 ? cancelledHeaderLine : lineCount
```

Comment:
I can't read nested ? : without layout structure.
I don't mind you removing brackets but please leave the layout.